dotCMS/src/main/java/com/dotcms/rendering/velocity/viewtools/content/StoryBlockMap.java

```diff
@@ -0,0 +1,44 @@
+package com.dotcms.rendering.velocity.viewtools.content;
+
+
+import com.dotcms.contenttype.transform.field.LegacyFieldTransformer;
+import com.dotcms.repackage.org.codehaus.jettison.json.JSONException;
+import com.dotcms.repackage.org.codehaus.jettison.json.JSONObject;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.contentlet.model.Contentlet;
+import com.dotmarketing.portlets.structure.model.Field;
+import com.liferay.util.StringPool;
+import org.apache.commons.lang.builder.ToStringBuilder;
+
+public class StoryBlockMap {
+
+    private String type = StringPool.BLANK;
```

Comment:
it seems it could be final seems do not have any set

Reply:
it's set over here https://github.com/dotCMS/core/pull/20785/files#diff-7aa369151c4492eaa1284343818cee85b0f7b92966ac27e6a14a58a4da69e0e0R23